radio/src/pulses/afhds3.h

```diff
@@ -0,0 +1,523 @@
+/*
```

Comment:
code style to review in this whole file

Reply:
Applied - please see attached eclipse style.
It seems it has small issues while handling marco definition around structs like PACKED.